packages/components/src/navigator/test/index.tsx

```diff
@@ -148,23 +147,6 @@ function CustomNavigatorBackButton( {
 	);
 }
 
-function CustomNavigatorToParentButton( {
-	onClick,
-	...props
-}: Omit< ComponentPropsWithoutRef< typeof NavigatorBackButton >, 'onClick' > & {
-	onClick?: CustomTestOnClickHandler;
-} ) {
-	return (
-		<NavigatorToParentButton
-			onClick={ () => {
-				// Used to spy on the values passed to `navigator.goBack`.
-				onClick?.( { type: 'goToParent' } );
-			} }
-			{ ...props }
-		/>
-	);
-}
```

Comment:
I removed this so that our tests don't use deprecated components, but I could also add them back and add an assertion to make sure that a warning message was logged to console

Reply:
Doesn't hurt to keep them, with a `deprecated` logging assertion as you suggested.

Reply:
Alright! Added some specific tests for deprecated APIs https://github.com/WordPress/gutenberg/pull/63317/commits/f1f688868137b9ff576cf00d71ee4fd9524f912b